### ISSUE TITLE
menuInsets not working

### ISSUE THREAD

**Issue by diilarakaymak** [rechsteiner/Parchment#573]
Hi, we were using the version 1.7.0 of your library in our application. We have updated your library to version 3.0.1 but "menuInsets" property not working correctly this version.

I also tried it on class CalendarViewController in your sample project and encountered the same error. Briefly, I get the same error when I try not only in class CalendarViewController but also in other classes.

I shared the line of code and the screenshot of your application.
pagingViewController.menuInsets = UIEdgeInsets(top: 15, left: 10, bottom: 15, right: 10)

<a href="https://ibb.co/Cvmk0TB"><img src="https://i.ibb.co/P4F7Cfx/Simulator-Screen-Shot-i-Phone-11-2021-05-03-at-11-55-47.png" alt="Simulator-Screen-Shot-i-Phone-11-2021-05-03-at-11-55-47" border="0" /></a>

How can I solve this?

**Comment by Scyano** [comment]
I have the same issue. and this is a bug, you can check the bad code below
```
open class PagingView: UIView {
  public var options: PagingOptions {
    didSet {
      heightConstraint?.constant = options.menuItemSize.height
      collectionView.backgroundColor = options.menuBackgroundColor
    }
  }
...
```

heightConstraint will be change to `options.menuItemSize.height` when we set options, it should change to
```
heightConstraint?.constant = options.menuHeight
```
which is a calculate property contains itemHeight and insets
```
public var menuHeight: CGFloat {
    return menuItemSize.height + menuInsets.top + menuInsets.bottom
  }
```

@diilarakaymak 
Maybe you can get the constraint by `pagingVC.collectionView.constaints`, and change it directly

**Comment by rechsteiner** [comment]
Thanks @Scyano! I'll look into getting this fixed soon

**Comment by ohmiya-masatsugu** [comment]
Thanks for @Scyano my temporary patch
```
final class MyViewController: PagingViewController {
    override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
        
        self.menuInsets = UIEdgeInsets(top: 100, left: 0, bottom: 0, right: 0)

        // https://github.com/rechsteiner/Parchment/issues/573
        if let pagingView = self.view as? PagingView {
            if let constraint = pagingView.constraints.first( where: {
                $0.firstItem as? NSObject == pagingView.collectionView
                && $0.firstAttribute == .height
                && $0.relation == .equal
            }) {
                constraint.constant =  menuItemSize.height + menuInsets.top + menuInsets.bottom
                self.view.setNeedsLayout()
            }
        }
    }
}
```

**Comment by rechsteiner** [comment]
This should be fixed now #619
